tests/tests_suite_test.go

```diff
@@ -87,7 +92,7 @@ func TestTests(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	SetDefaultEventuallyTimeout(10 * time.Second)
+	SetDefaultEventuallyTimeout(60 * time.Second)
```

Comment:
I don't think I'm in favor of increasing this default timeout.  While the effects are less pronounced when running in parallel, it really slows down the sequential run.  Since we're relying on the sequential run for the time being, I vote we keep the previous 10 sec default and revisit at a later date.